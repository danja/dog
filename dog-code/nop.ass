START 0000
NOP
HALT
