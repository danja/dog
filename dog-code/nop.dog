start
0105
00 NOP
FF HALT
