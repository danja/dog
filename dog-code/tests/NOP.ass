00
00
TEST
NOP ; do nothing 5 times
NOP
NOP
NOP
DUMP
