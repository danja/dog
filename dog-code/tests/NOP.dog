F9 TEST
00 NOP ; do nothing 5 times
00 NOP
00 NOP
00 NOP
F8 DUMP
