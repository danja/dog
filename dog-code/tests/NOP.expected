{
"pc": 5,
"status": -1,
"accA": -1,
"accB": -1,
"xReg": -1,
"pcStackP": -1,
"xStackP": -1
}
